package.json

```diff
@@ -37,13 +37,15 @@
   },
   "devDependencies": {
     "@types/chai": "^4.2.5",
+    "@types/chai-as-promised": "^7.1.8",
     "@types/express": "^4.17.19",
     "@types/mocha": "^5.2.7",
     "@types/node": "^20.8.5",
     "@types/node-fetch": "^2.5.3",
     "@typescript-eslint/eslint-plugin": "^4.8.2",
     "@typescript-eslint/parser": "^4.8.2",
-    "chai": "^4.2.0",
+    "chai": "^4.3.10",
+    "chai-as-promised": "^7.1.1",
```

Comment:
Please don't use the "chai-as-promised" plugin (actually, it already seems unused). Now that we have async / await, you can just do `expect(await whatever)...` instead of `expect(whatever).to.eventually...`. It's simpler to use the language directly instead of hacking promise-handling magic into Chai.